pom.xml

```diff
@@ -39,8 +39,8 @@
 	<properties>
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
-		<spring-cloud-release.version>Finchley.RC2</spring-cloud-release.version>
-		<spring-boot-release.version>2.0.2.RELEASE</spring-boot-release.version>
+		<spring-cloud-release.version>Finchley.RELEASE</spring-cloud-release.version>
```

Comment:
You know, according the architecture of the Spring Cloud train, we can't rely on the Spring Cloud BOM, just because eventually we are going to be a part of that train #425.

Therefore we need to consider some other solution like use BOM from the particular Spring Cloud dependencies, for example for the `spring-cloud-context` we have to use `spring-cloud-commons-dependencies`: https://github.com/spring-cloud/spring-cloud-aws/pull/347 and so.

Also we should consider somehow do not use `spring-boot` version explicitly. I somehow believe that `spring-cloud-build` parent does the trick for us

Reply:
I added #773 regarding your first comment.
I'd rather not handle this right now though.
The latest `spring-cloud-build` specifies version 2.0.2 of Spring Boot, which is not latest.

Reply:
Re. Spring Boot version: well, unfortunate, agreed,. but that's not the way all other Spring Cloud projects go...
So, if we are going to be a part of the Spring Cloud Train anyway, we have to follow rules there and be aligned with dependencies are dictated from there. 🤷‍♂️

Reply:
Looks like we don't even need to specify the boot version. So, I removed that altogether.